mathesar_ui/src/component-library/dropdown/Dropdown.svelte

```diff
@@ -40,6 +40,7 @@
   $: tgClasses = calculateTriggerClass(triggerClass, showArrow);
 
   function toggle(e: Event) {
+    e.preventDefault();
```

Comment:
To make it work when wrapped with an anchor tag. Since the event is already not bubbling up and this component will always necessarily handle the click event, preventing the default behaviour should be fine